tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,846 @@
+# | :ref:`tutorials-introductory`| :doc:`/tutorials/introductory/customizing` |
+# +------------------------------+--------------------------------------------+
+# | :ref:`tutorials-intermediate`| :doc:`/tutorials/intermediate/legend_guide`|
+# +------------------------------+--------------------------------------------+
+# | :ref:`tutorials-colors`      | :doc:`/tutorials/colors/colors`            |
+# |                              +--------------------------------------------+
+# |                              | :doc:`/tutorials/colors/colormaps`         |
+# +------------------------------+--------------------------------------------+
+# | :ref:`tutorials-text`        | :doc:`/tutorials/text/text_intro`          |
+# |                              +--------------------------------------------+
+# |                              | :doc:`/tutorials/text/text_props`          |
+# |                              +--------------------------------------------+
+# |                              | :doc:`/tutorials/text/annotations`         |
+# +------------------------------+--------------------------------------------+
+#
+# For complete information about available methods for Artists, refer to the
+# table below.
+#
+# +------------------------------------+------------------------------------+
+# | Explicit                           | Implicit                           |
+# +====================================+====================================+
+# | :class:`matplotlib.axes.Axes`      | :mod:`matplotlib.pyplot`           |
+# +------------------------------------+------------------------------------+
+#
+#
+# Pie Chart Examples
+# ------------------
+#
+# Matplotlib pie charts create wedges based on data. They manipulate the size
+# of the Artists based on the ratio of the wedge to the sum of the data. The
+# ``.pie()`` method is similar in both explicit and implicit approaches.
+#
+# The code below illustrates various levels of configuration in keyword
+# arguments as well as Artist methods for both explicit and implicit
+# programming.
+#
+
+# Sample data for monthly spending averages.
+
+budget = [475, 300, 125, 50]
+# Data points correspond to wedge size as a ratio of total sum.
+# Matplotlib methods calculate these values automatically based on input.
+
+descriptions = ['Shared house\nin Philadelphia',
+                'Dog costs, phone,\nutilities',
+                'Groceries\n& takeout',
+                'Treasury bonds']
+categories = ['Rent', 'Bills', 'Food', 'Savings']
+# Lists of strings contribute to labeling corresponding data.
+
+colors = ['#1f77b4', '#ff7f0e', '#d62728', '#2ca02c']
+# Hex color codes determine respective wedge color.
+
+explode = [0, 0.1, 0.15, 0.35]
+# List of floats represents percentage of radius to separate from center.
+
+
+def autopct_format(percent, group):
+    """
+    Takes percent equivalent and calculates original value from data.
+    Returns fstring of value new line above percentage.
```

Comment:
It returns a string; an f-string is a different type of _literal_, but it still results in a string.